<!DOCTYPE html>
<html lang="en">
<head>
   <meta charset="utf-8">
   <title> Homepage of Dr. Suoqin Jin @ Wuhan University - Home</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta name="description" content="Dr. Suoqin Jin at the School of Mathematics and Statistics, Wuhan University">
   <meta name="author" content="">
   <!-- Le styles -->
   <link href="css/bootstrap.min.css" rel="stylesheet">
   <link href="css/bootstrap-responsive.min.css" rel="stylesheet">
   <link href="css/theme.css" rel="stylesheet">
</head>


<body>
<div class="container">
   <header class="jumbotron subhead" id="overview">
      <p class="lead"> </p>
      <h1> Homepage of Prof. Suoqin Jin @ Wuhan University </h1>
   </header>
   <div class="masthead">
      <div class="navbar">
         <div class="navbar-inner">
            <div class="container">
               <ul class="nav">
                 <li><a href="index.html">Home</a></li>
                  <li><a href="research.html">Research</a></li>
             <!--    <li><a href="people.html">People</a></li> -->
                  <li><a href="publications.html">Publications</a></li>
            <!--       <li><a href="resources.html">Resources</a></li> -->
                 <li class="active" ><a href="#">Resources</a></li>
             <!--     <li><a href="jobs.html">Jobs</a></li> -->
                 <li><a href="contact.html">Contact</a></li>

               </ul>
            </div>
         </div>
      </div>
   </div>

         <hr>
<div id="media">
            <div class="row-fluid post_column">
               <div class="span12 post">
                  <div class="text">
                     <h4><a href="https://github.com/jinworks/CellChat"> CellChat </a></h4>
                   </div>
                  <div class="img">
                     <a href="https://github.com/jinworks/CellChat">
                     <img src="images/CellChat_Logo.png" class="float-right" alt="" style="width: 200px; height: auto;">
                     </a>
                  </div>
                  <div class="text">
                     <br/>
                     <p style="font-size:16px"> CellChat is an <b>R toolkit</b> for inference, visualization and analysis of <b>cell-cell communication</b> from single-cell data. <br>
                       CellChat requires gene expression data of cells as the user input and models the probability of cell-cell communication by integrating gene expression with CellChatDB, which contains prior knowledge of the interactions between signaling ligands, receptors and their cofactors.</br>
                       Upon infering the intercellular communication network, CellChat provides functionality for further data exploration, analysis, and visualization. </p>
                  </div>

                   <hr>

                  <div class="text">
                     <h4><a href="http://www.cellchat.org/"> CellChat Explorer </a></h4>
                   </div>
                  <div class="img">
                     <a href="http://www.cellchat.org/">
                     <img src="images/CellChatDB_Logo.png" class="float-right" alt="" style="width:200px; height: auto;">
                     </a>
                  </div>
                  <div class="text" style="font-size:16px">
                     <br/>
                     <p
                     <ul>
                      The user-friendly web-based “CellChat Explorer” that contains two major components:
                       <li> <b>Ligand-Receptor Interaction Explorer </b>that allows easy exploration of our novel ligand-receptor interaction database, a comprehensive recapitulation of known molecular compositions including multimeric complexes and co-factors. Our database CellChatDB is a manually curated database of literature-supported ligand-receptor interactions in both <b>human</b> and <b>mouse</b>.</li>
                       <li><b>Cell-Cell Communication Atlas Explorer </b> that allows easy exploration of the cell-cell communication for any given scRNA-seq dataset that has been processed by our R toolkit CellChat.</li>
                       <li>A standalone <b>CellChat Shiny App </b>for our Cell-Cell Communication Atlas Explorer.</li>
                       </ul>
                       </p>
                  </div>
                  <hr>
                  
                  <div class="text">
                     <h4><a href="https://github.com/jinworks/scAI"> scAI </a></h4>
                   </div>
                  <div class="img">
                     <a href="https://github.com/jinworks/scAI">
                     <img src="images/scAI-logo.png" class="float-right" alt="" style="width: 200px; height: auto;">
                     </a>
                  </div>
                  <div class="text">
                     <br/>
                     <p style="font-size:16px"> scAI is an R tool for <b>integrative analysis of single-cell multi-omics data</b>. <br>
                     scAI is an unsupervised approach for integrative analysis of gene expression and chromatin accessibility or DNA methylation proflies measured in the same individual cells.</br>
                     Once the single cell multi-omics data are decomposed into multiple biologically relevant factors, the package provides functionality for further data exploration, analysis, and visualization. </p>
                  </div>
                  <hr>
                  
                  <div class="text">
                     <h4><a href="https://github.com/jinworks/scEpath"> scEpath </a></h4>
                   </div>
                  <div class="img">
                     <a href="https://github.com/jinworks/scEpath">
                     <img src="images/scEpath_landscape.png" class="float-right" alt="" style="width: 200px; height: auto;">
                     </a>
                  </div>
                  <div class="text">
                     <br/>
                     <p style="font-size:16px"> scEpath is an energy landscape-based approach for <b>measuring developmental states and inferring cellular trajectories</b> from single cell RNA-seq data. <br>
                     scEpath is a novel computational method for quantitatively measuring developmental potency and plasticity of single cells and transition probabilities between cell states, and inferring lineage relationships and pseudotemporal ordering from single-cell gene expression data.</br>
                     In addition, scEpath performs many downstream analyses including identification of the most important marker genes or transcription factors for given cell clusters or over pseudotime. </p>
                  </div>



                   <br/>
               </div>
            </div>


          </div>
         </div>


         <!-- Javascript files -->
         <script src="js/jquery-1.9.1.min.js"></script>
         <script src="js/bootstrap.min.js"></script>
         <script> $('#main-carousel').carousel(); </script>
      </body>
   </html>

